data-serving/scripts/convert_data.py

```diff
@@ -0,0 +1,55 @@
+# Lint as: python3
+'''Script to convert CSV line-list data into json compliant with the MongoDB schema.'''
+
+import argparse
+import csv
+import logging
+import json
+
+parser = argparse.ArgumentParser(
+    description='Convert CSV line-list data into json compliant with the MongoDB schema.')
+parser.add_argument('--csv_path', required=True)
+parser.add_argument('--json_path', default='cases.json')
+
+
+def main():
+    logging.basicConfig(filename='convert_data.log', level=logging.INFO)
+    args = parser.parse_args()
+    logging.info("Args: %s", args)
+
+    print('Reading data from', args.csv_path)
+    cases = read_csv(args.csv_path)
+
+    print('Converting data to new schema')
+    convert(cases)
+
+    print('Writing results to', args.json_path)
+    write_json(cases, args.json_path)
+
+    print('Great success! 🎉')
+
+
+def read_csv(csv_path):
+    cases = []
+    with open(csv_path, newline='') as csvfile:
+        csvreader = csv.DictReader(csvfile)
+        logging.debug('fields: %s', csvreader.fieldnames)
+        for record in csvreader:
```

Comment:
Using for loops in this file is probably fine. If it gets slow (there's a fair amount of total data), I'd switch to reading the csv in with pandas and operating on it as a dataframe, or using the `.values` therefrom to work with numpy. The vectorized functions will be ~thousands of times faster than for loops.

Reply:
much faster -- and cleaner! thanks!